tests/simd/scalar_ops.ml

```diff
@@ -29,3 +29,18 @@ module Int64x2 = struct
     eq (int64x2_low_int64 c3) (int64x2_high_int64 c3) 48L 0L
   ;;
 end
+
+module Int64 = struct
+
+  type t = int64
+
+  external bit_deposit : t -> t -> t = "caml_vec128_unreachable" "caml_bmi2_int64_deposit_bits"
+      [@@noalloc] [@@unboxed] [@@builtin]
+  external bit_extract : t -> t -> t = "caml_vec128_unreachable" "caml_bmi2_int64_extract_bits"
+      [@@noalloc] [@@unboxed] [@@builtin]
+
+  let () =
+    eq (bit_deposit 3L 4L)     (bit_extract 3L 4L)     0x4L 0x0L;
+    eq (bit_deposit 235L 522L) (bit_extract 235L 522L) 0xAL 0x3L;
```

Comment:
(The way the test is expressed feels a
bit odd: on each line the tests of
`bit_deposit` and `bit_extract` are
actually independent.)

Reply:
True - split them up